data/gui/window/addon_auth.cfg

```diff
@@ -95,6 +95,26 @@
 
 			[/row]
 
+			[row]
+				grow_factor = 1
+
+				[column]
+					grow_factor = 1
+					horizontal_alignment = "left"
+					border = "all"
+					border_size = 5
+
+					[toggle_button]
+						id = "remember_password"
+						definition = "default"
+
+						label = _ "Save password locally (encrypted)"
```

Comment:
> campaignd has always called it "passphrase", and that's what it's in the _server.pbl.

But the new UI string says otherwise, unless it's this way to reuse a string from elsewhere.

Reply:
https://github.com/wesnoth/wesnoth/blob/master/data/gui/window/mp_login.cfg#L199